provider/cmd/pulumi-resource-xyz/package.json

```diff
@@ -3,11 +3,11 @@
     "bin": "bin/index.js",
     "dependencies": {
         "@pulumi/pulumi": "^3.38.0",
-        "@pulumi/aws": "^5.0.0"
+        "@pulumi/aws": "^6.32.0"
     },
     "devDependencies": {
-        "@types/node": "^10.0.0",
-        "pkg": "^5.6.0",
-        "typescript": "^4.6.3"
+        "@types/node": "^20.12.11",
+        "@yao-pkg/pkg": "^5.11.5",
```

Comment:
I'm not totally sold that this is a very credible alternative to the archived `@vercel/pkg`; it looks like it's kind of a one person project. 

That said, I'm not going to block the change since the situation with the existing dependency is not great either.